apps/studio/src/components/VersionWrapper/VersionModal.tsx

```diff
@@ -32,7 +32,7 @@ export const VersionModal = ({ isOpen, onClose }: VersionModalProps) => {
       <ModalOverlay />
       <ModalContent>
         <ModalCloseButton size="lg" />
-        <ModalHeader>Update Available</ModalHeader>
+        <ModalHeader ml="3.5rem">Update Available</ModalHeader>
```

Comment:
double checking here if this should be `mr`

Reply:
yup you are right, thanks for spotting
have fixed 👍